main/AppCommon.c

```diff
@@ -656,26 +656,28 @@ STATUS app_common_createStreamingSession(PAppConfiguration pAppConfiguration, PC
     CHK_STATUS(
         (rtp_transceiver_onBandwidthEstimation(pStreamingSession->pVideoRtcRtpTransceiver, (UINT64) pStreamingSession, app_common_onBandwidthEstimation)));
 
+#ifndef KVS_PLAT_ESP_FREERTOS
```

Comment:
Please separate this two ways audio as one independent PR before this ANYAKE PR.